fava/api/__init__.py

```diff
@@ -365,16 +366,24 @@ def is_valid_document(self, file_path):
         """Check if file_path is the path to a document.
 
         It should either occur in one of the Document entries or in a
-        "statement"-metadata in a Transaction entry.
+        "statement"-metadata in a Transaction entry or one of it's Postings.
```

Comment:
We should restrict to documents. For each statement, there should exist a (generated) document entry.